docfiles/docs.js

```diff
@@ -477,7 +477,7 @@ function setStickyColumn() {
                     scrollTop: $(heading).offset().top
                 }, 1000);
                 if (window.navigator.userAgent.indexOf('Edge') !== -1) {
-                    heading.scrollIntoView();
+                    heading.nextElementSibling.scrollIntoView(false);
```

Comment:
You could test that nextElemenySibling exists and scroll into view too

Reply:
I found a better solution adding a few Edge CSS specific styles :)